langwatch/src/app/api/scenario-events/[[...route]]/mappings.ts

```diff
@@ -0,0 +1,65 @@
+/**
+ * Scenario Event Mappings for Elasticsearch
+ * Maps the schema defined in schemas.ts to Elasticsearch field types
+ */
+
+import type { MappingProperty } from "@elastic/elasticsearch/lib/api/types";
+
+// Base event mapping (common to all events)
+const baseScenarioEventMapping = {
+  type: { type: "keyword" },
+  timestamp: { type: "date" },
+  rawEvent: { type: "object" }, // Added from base schema
```

Comment:
no object, use FLATENNED_TYPE here too, and generally anything with `raw` on it, as any possible union type on AG might break us and I don't think we need to index the inner of the events, we already have the `type` up above, this will allow us for a lot of queries already